2022-04-21 14:53:43,450 INFO  ==============================================================
2022-04-21 14:53:43,451 INFO  Id:    SoftwareManagement.sm_docker_install_remove_multiple
2022-04-21 14:53:43,452 INFO  Dir:   software_management_end_to_end/
2022-04-21 14:53:43,453 INFO  Title: Validate end to end behaviour for the docker plugin for multiple docker images with mixed versions
2022-04-21 14:53:43,454 INFO  ==============================================================
2022-04-21 14:53:43,464 INFO  Using sw id database: {'asciijump': '5475369', 'robotfindskitten': '5474869', 'squirrel3': '5475279', 'rolldice': '5152439', 'moon-buggy': '5439204', 'apple': '5495053', 'banana': '5494888', 'cherry': '5495382', 'watermelon': '5494510', 'registry': '8018911', 'hello-world': '8021526', 'docker/getting-started': '8021973', 'alpine': '7991792'}
2022-04-21 14:53:43,465 INFO  EnvironmentC8y Setup
2022-04-21 14:53:44,077 INFO  Executed systemctl<serv_mapper1>, exit status 3
2022-04-21 14:53:44,188 INFO  Restarting mosquitto too frequently in the last 10 seconds. It was only up for 5 seconds
2022-04-21 14:53:44,189 INFO  Delaying execution by 6 seconds
2022-04-21 14:54:04,452 INFO  Executed sudo<tedge_connect_c8y>, exit status 0, duration 14 secs
2022-04-21 14:54:05,067 INFO  Executed sudo<tedge_connect_c8y_test>, exit status 0
2022-04-21 14:54:05,653 INFO  Executed systemctl<serv_mosq2>, exit status 0
2022-04-21 14:54:06,373 INFO  Executed systemctl<serv_mapper3>, exit status 0
2022-04-21 14:54:08,593 INFO  State of current operation: SUCCESSFUL
2022-04-21 14:54:08,822 INFO  Response status: 201
2022-04-21 14:54:08,823 INFO  Response to action: {
    "creationTime": "2022-04-21T14:54:08.778Z",
    "deviceName": "mythicd",
    "deviceId": "17547554",
    "self": "https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/17588923",
    "id": "17588923",
    "status": "PENDING",
    "description": "Apply software changes, triggered from PySys: [{'action': 'install', 'id': '8021526', 'name': 'hello-world', 'url': '', 'version': '::docker'}, {'action': 'install', 'id': '8018911', 'name': 'registry', 'url': '', 'version': '2.6.2::docker'}]",
    "c8y_SoftwareUpdate": [
        {
            "name": "hello-world",
            "action": "install",
            "id": "8021526",
            "version": "::docker",
            "url": ""
        },
        {
            "name": "registry",
            "action": "install",
            "id": "8018911",
            "version": "2.6.2::docker",
            "url": ""
        }
    ]
}
2022-04-21 14:54:08,824 INFO  Started operation: {'creationTime': '2022-04-21T14:54:08.778Z', 'deviceName': 'mythicd', 'deviceId': '17547554', 'self': 'https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/17588923', 'id': '17588923', 'status': 'PENDING', 'description': "Apply software changes, triggered from PySys: [{'action': 'install', 'id': '8021526', 'name': 'hello-world', 'url': '', 'version': '::docker'}, {'action': 'install', 'id': '8018911', 'name': 'registry', 'url': '', 'version': '2.6.2::docker'}]", 'c8y_SoftwareUpdate': [{'name': 'hello-world', 'action': 'install', 'id': '8021526', 'version': '::docker', 'url': ''}, {'name': 'registry', 'action': 'install', 'id': '8018911', 'version': '2.6.2::docker', 'url': ''}]}
2022-04-21 14:54:11,037 INFO  State of operation 17588923 : PENDING
2022-04-21 14:54:13,258 INFO  State of operation 17588923 : EXECUTING
2022-04-21 14:54:15,485 INFO  State of operation 17588923 : EXECUTING
2022-04-21 14:54:17,698 INFO  State of operation 17588923 : EXECUTING
2022-04-21 14:54:19,928 INFO  State of operation 17588923 : FAILED
2022-04-21 14:54:19,929 ERROR Stopping as the operation has failed
2022-04-21 14:54:19,930 WARN  SystemError occurred while running test - The operation has failed
Traceback (most recent call last):
  File "/home/pi/env-pysys/lib/python3.7/site-packages/pysys/baserunner.py", line 1292, in __call__
    self.testObj.setup()
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/sm_docker_install_remove_multiple/run.py", line 55, in setup
    self.wait_until_succcess()
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/environment_sm_management.py", line 324, in wait_until_succcess
    self.wait_until_status("SUCCESSFUL")
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/environment_sm_management.py", line 361, in wait_until_status
    raise SystemError("The operation has failed")
SystemError: The operation has failed
2022-04-21 14:54:19,953 WARN  SystemError: The operation has failed ... blocked 
2022-04-21 14:54:19,954 INFO  
2022-04-21 14:54:19,955 INFO  cleanup:
2022-04-21 14:54:23,879 INFO  Executed sudo<tedge_disconnect_c8y>, exit status 0
2022-04-21 14:54:24,442 INFO  Executed systemctl<serv_mapper5>, exit status 3
2022-04-21 14:54:24,446 INFO  
2022-04-21 14:54:24,447 INFO  Test duration: 41.01 secs
2022-04-21 14:54:24,448 INFO  Test final outcome:  BLOCKED
2022-04-21 14:54:24,449 INFO  Test outcome reason: SystemError: The operation has failed
2022-04-21 14:54:24,450 INFO  
